src/test/java/net/snowflake/client/log/JDK14LoggerWithClientLatestIT.java

```
@@ -93,4 +96,62 @@ public void testJDK14LoggerWithQuotesInMessage() {
     JDK14Logger.setLevel(Level.OFF);
   }
+
+  @Test
+  public void testJDK14LoggingWithMissingLogPathClientConfig() {
+    Path configFilePath = Paths.get("config.json");
+    String configJson = "{\"common\":{\"log_level\":\"debug\"}}";
+
+    Path homeLogPath = Paths.get(homePath, "jdbc");
+    try {
+      Files.write(configFilePath, configJson.getBytes());
+      Properties properties = new Properties();
+      properties.put("client_config_file", configFilePath.toString());
+      Connection connection = getConnection(properties);
+      connection.createStatement().executeQuery("select 1");
+
+      File file = new File(homeLogPath.toString());
+      assertTrue(file.exists());
+    } catch (IOException e) {
+      fail("testJDK14LoggingWithMissingLogPathClientConfig failed");
+    } catch (SQLException e) {
+      fail("testJDK14LoggingWithMissingLogPathClientConfig failed");
+    } catch (Exception e) {
+      fail("testJDK14LoggingWithMissingLogPathClientConfig failed");
+    } finally {
+      try {
+        Files.deleteIfExists(configFilePath);
+        FileUtils.deleteDirectory(new File(homeLogPath.toString()));
+      } catch (IOException e) {
+        fail("testJDK14LoggingWithMissingLogPathClientConfig failed");
+      }
+    }
+  }
+
+  @Test
+  public void testJDK14LoggingWithMissingLogPathNoHomeDirClientConfig() {
+    System.clearProperty("user.home");
+
+    Path configFilePath = Paths.get("config.json");
+    String configJson = "{\"common\":{\"log_level\":\"debug\"}}";
+    try {
+      Files.write(configFilePath, configJson.getBytes());
+      Properties properties = new Properties();
+      properties.put("client_config_file", configFilePath.toString());
+      Connection connection = getConnection(properties);
+
+      fail("testJDK14LoggingWithMissingLogPathNoHomeDirClientConfig failed");
+    } catch (IOException e) {
+      fail("testJDK14LoggingWithMissingLogPathNoHomeDirClientConfig failed");
```

let's not catch other exceptions and let the test fail